BackofficeBundle/Tests/Functional/Controller/NodeControllerTest.php

```diff
@@ -104,6 +108,12 @@ public function testNewNodePageHome()
 
         $this->assertEquals($nbLink + 2, $crawler->filter('a')->count());
 
+        $this->redirectionRepository = static::$kernel->getContainer()->get('open_orchestra_model.repository.redirection');
+        $this->routeDocumentRepository = static::$kernel->getContainer()->get('open_orchestra_model.repository.route_document');
+
+        $this->assertEquals(1, count($this->redirectionRepository->findAll()));
+        $this->assertEquals(62, count($this->routeDocumentRepository->findAll()));
```

Comment:
Why not? AnyWay, I'm changing the status of the created node to publish in order to test new redirection and route counts. Online in few minutes...